group_vars/all

```diff
@@ -351,3 +351,10 @@
   gpfs_groups:
     - {"name": "gpfs4", "authorized_keys":"/gpfs4/data/user/home/$DOWNSTREAM_USER/.ssh/authorized_keys", "host": "login001", "private_key":"/gpfs4/data/user/home/$DOWNSTREAM_USER/.ssh/id_ecdsa"}
     - {"name": "gpfs5", "authorized_keys":"/gpfs5/data/user/home/$DOWNSTREAM_USER/.ssh/authorized_keys", "host": "login002", "private_key":"/gpfs5/data/user/home/$DOWNSTREAM_USER/.ssh/id_ecdsa"}
+
+# cod_slurm
+  frozen_file_list:
+    - /cm/shared/apps/slurm/var/etc/cgroup.conf
+    - /cm/shared/apps/slurm/var/etc/slurm.conf
```

Comment:
Path should be: `/cm/shared/apps/slurm/var/etc/slurm/slurm.conf`

Reply:
Resolved d260d795b73ea4468ac0fe7dd3787567730fe8c4